<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="pagetTeacher">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="tableWidgetTeacher">
            <column>
             <property name="text">
              <string>编号</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>姓名</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>性别</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>出生年月</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>职称</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>最后学历</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>工资</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>婚否</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>修改</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>删除</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>课程信息</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonAddTeacher">
              <property name="text">
               <string>添加教师</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonExit">
              <property name="text">
               <string>退出</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageClasses">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableWidget" name="tableWidgetClasses">
            <column>
             <property name="text">
              <string>授课星期</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>授课课时</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>授课班级</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>修改</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>删除</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonAddTeachingInformation">
              <property name="text">
               <string>添加授课信息</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonBack">
              <property name="text">
               <string>返回</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>教师信息管理系统</string>
    </property>
    <addaction name="actionOpen_file"/>
    <addaction name="actionSave_file_as"/>
    <addaction name="actionSave_file"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>筛选</string>
    </property>
    <addaction name="actionId"/>
    <addaction name="actionSex"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionSave_file_as">
   <property name="text">
    <string>Save as file</string>
   </property>
  </action>
  <action name="actionSave_file">
   <property name="text">
    <string>Save file</string>
   </property>
  </action>
  <action name="actionId">
   <property name="text">
    <string>Id</string>
   </property>
  </action>
  <action name="actionSex">
   <property name="text">
    <string>Sex</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
